cmake/acadosConfig.cmake.in

```diff
@@ -3,12 +3,13 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")     # for FindOpenBLA
 
 find_package(blasfeo)
 find_package(hpipm)
+find_package(qpOASES_e)
```

Comment:
`qpOASES` is not a dependency, better move it back with the other optional QP solvers

Reply:
I was adding it to the defaults because it was used in the tests.
I will remove it from the default values, should I keep it among the tests run by travis or similar?

Reply:
What about keeping qpOASES ON on travis (so all tests are run), but OFF by default in cmake?

Reply:
ok, qpOASES is in the tests but is not included in acados by default